utils/install-protobuf.sh

```diff
@@ -0,0 +1,9 @@
+PROTOBUF_VERSION=3.16.0
+git clone -b v${PROTOBUF_VERSION} --recursive https://github.com/google/protobuf.git
```

Comment:
Would you update the windows version of this script to also use the tag instead of the specific commit

Reply:
Done